playground/js-sourcemap/foo-with-sourcemap.js

```diff
@@ -1,5 +1,3 @@
 export const foo = 'foo'
 
 // default boundary sourcemap with magic-string
```

Comment:
I think it's better to inject this comment together with the sourcemap comment since this describes the sourcemap comment.